yt_dlp/extractor/youtube.py

```diff
@@ -69,7 +69,15 @@
 )
 
 STREAMING_DATA_CLIENT_NAME = '__yt_dlp_client'
-STREAMING_DATA_PO_TOKEN = '__yt_dlp_po_token'
+STREAMING_DATA_INITIAL_PO_TOKEN = '__yt_dlp_po_token'
+STREAMING_DATA_FETCH_PO_TOKEN = '__yt_dlp_fetch_po_token'
```

Comment:
> `STREAMING_DATA_FETCH_PO_TOKEN = '__yt_dlp_fetch_po_token'`

The naming threw me off at first. Can we use names that make it more clear they point to a function?
```suggestion
STREAMING_DATA_PO_TOKEN_FUNC = '__yt_dlp_po_token_func'
```

or something?